.github/workflows/release-tag.yml

```diff
@@ -285,20 +278,13 @@ jobs:
       run: | # sh
 
         if [[ "${{ matrix.edition }}" == "ee" ]]; then
-          git tag -f ${{ inputs.tag_name }}-ee ${{ needs.check-version.outputs.ee }}
+          git tag -f ${{ inputs.tag_name }}-ee ${{ needs.check-version.outputs.oss }}
```

Comment:
ok, this was a little bug 😳 

https://github.com/metabase/metabase/actions/runs/12349125210/job/34459591755